CMakeLists.txt

```diff
@@ -373,7 +373,7 @@ if (NOT EMSCRIPTEN)
   endif ()
 
   # test running
-  find_package(PythonInterp 2.7 REQUIRED)
+  find_package(PythonInterp 3.4 REQUIRED)
```

Comment:
A better change would be:
``` diff
-    find_package(PythonInterp 2.7 REQUIRED)
+    find_package(Python COMPONENTS Interpreter 3.4 REQUIRED)
```
or even

``` diff
-    find_package(PythonInterp 2.7 REQUIRED)
+    find_package(Python3 COMPONENTS Interpreter 3.4 REQUIRED)
```

See https://cmake.org/cmake/help/latest/module/FindPython.html and https://cmake.org/cmake/help/latest/module/FindPython3.html .

Reply:
I think I tried that and ran into some issues.  Can do leave that for a followup?